README.md

```diff
@@ -55,6 +55,7 @@ The inputs this action uses are:
 | `create_redis` | `false` | None | True to create a Redis instance for the app. |
 | `create_postgres` | `false` | None | True to create a Postgres instance for the app. |
 | `create_persistent_filesystem` | `false` | None | True to create a persistent filesystem for the app. |
+| `env_vars` | `false` | None | Environment variables to add. Separate multiple variables with a space. Example: `env_vars: "FOO=BAR FOO_SECRET=${{ secrets.FOO_SECRET }}"` |
```

Comment:
Will this work without quotation marks?

Reply:
With the current design, no. Without string, it would not be possible to split them. But er cam change it

To make it more transparent/less error prone, we can do this:
```
name: Deploy to Dash Enterprise

on:
  push:
    branches:
      - main

jobs:
  deploy:
    name: 'Deploy to Dash Enterprise'
    runs-on: ubuntu-latest
    strategy:
      matrix:
        env_var:
          - name: "FOO"
            value: "BAR"
          - name: "FOO_SECRET"
            value: "${{ secrets.FOO_SECRET }}"

  
```
Or 
```
name: Deploy to Dash Enterprise

on:
  push:
    branches:
      - main

jobs:
  deploy:
    name: 'Deploy to Dash Enterprise'
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: plotly/de-deploy@add-adding-env-variables
        with:
           env_vars: FOO=BAR FOO_SECRET=${{ secrets.FOO_SECRET }}
```
Or 
```
name: Deploy to Dash Enterprise

on:
  push:
    branches:
      - main

jobs:
  deploy:
    name: 'Deploy to Dash Enterprise'
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: plotly/de-deploy@add-adding-env-variables
        with:
          env_vars: |
            FOO=BAR
            FOO_SECRET=${{ secrets.FOO_SECRET }}
```
(Note that | must be there)

What do you think?